postinstall/post_programs_installer.sh

```diff
@@ -1,28 +1,35 @@
 #!/bin/bash
+
+#############################################################
 #
+# Author      : dimos katsimardos
+# Date        : 10/05/2020 | Bug Year
+# Description : Just a helper script to be run after
+#               installing Arch Linux in order to install
+#               some basic programs for basic stuff
 #
-# Archon -- Ελληνικός Arch Linux Installer
-# Copyright (c)2017 Vasilis Niakas, Salih Emin and Contributors
-#
-# This program is free software; you can redistribute it and/or modify
-# it under the terms of the GNU General Public License as published by
-# the Free Software Foundation version 3 of the License.
-#
-# Please read the file LICENSE, README and AUTHORS for more information.
+# Version 1.0 : Option to install music player, media player,
+#               Internet Explorer, Email Client, Text editor
 #
+# NOTE : If this program does not run after sftp from Win PC
+#        to VM for example, install dos2unix utility, via
+#        'sudo pacman -S dos2unix'. Also, if bash is not
+#        installed in the DE, for example Deepin Desktop, then
+#        also install bash via 'sudo pacman -S bash'.
 #
+#############################################################
+
 
 # Exit codes για success || failure
 #
 # setfont gr928a-8x16.psfu
 OK=0
 NOT_OK=1
-INTERNET_CONNECTION_ERROR=2
```

Comment:
Μην αλλάζεις τους κωδικούς σφαλμάτων όπως αυτά παρέχονται απο το POSIX standard. Μην τα αποθηκεύεις σε variable για να είναι εύκολη η ανάγνωση του κώδικα.

Reply:
Just made a merge cleanup and added header in post actions script ; conduct a review and then i could open a pull request for latest version if it is ok.